hello-signing.md

```diff
@@ -0,0 +1,132 @@
+# Hello World for Notation
+
+To get started with Notation, the most basic steps involve:
+
+1. Acquire the Notation CLI
+2. Generate a Private Key
+3. Sign an Artifact
+4. Verify an Artifact with a Notary Signature
+
+This document outlines progressive scenarios enabling a range of scenarios from the first use to a production deployment.
+
+## Scenarios
+
+To demonstrate how to store and sign a set of supply chain artifacts, we will walk through a set of scenarios:
+- Sign a single container image
+- Publish the container image across public registries, 
+- Import public content to a private registry
+- Promote from dev through production
+
+![](../media/notary-e2e-scenarios.svg)
+
+To illustrate these scenarios we will introduce two example companies:
+
+- Wabbit Networks is a small software company that produces network monitoring software.
+- ACME Rockets wishes to acquire network monitoring software.
+- ACME Rockets doesn't know about Wabbit Networks, but finds their [net-monitor software in Docker Hub](https://hub.docker.com/r/wabbitnetworks/net-monitor)
+- Since they've never heard of Wabbit Networks, they're a little reluctant to run network software without some validations.
+- ACME Rockets has a policy to only import Docker Hub certified software, or other approved vendors.
+- Wabbit Networks works with Docker Hub to get certified, to help with their customer confidence.
+- ACME Rockets will only deploy software that's been scanned and approved by the ACME Rockets security team. They know it's been approved because all approved software has been signed by the ACME Rockets security team.
+
+## Getting Started
+- Setup a few environment variables.  
+  >Note see [Simulating a Registry DNS Name](#simulating-a-registry-dns-name) to use `registry.wabbit-networks.io`
+  ```bash
+  export PORT=5000
+  export REGISTRY=localhost:${PORT}
+  export REPO=${REGISTRY}/net-monitor
+  export IMAGE=${REPO}:v1
+  ```
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for local docker operations
+- Run a local instance of the [CNCF Distribution Registry][cncf-distribution]
+  ```bash
+  docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Acquire the Notation CLI  
+Notation releases can be found at: [Notation Releases][notation-releases]  
+  ```bash
+  curl -O ~/bin/notation https://github.com/notaryproject/notation/releases/download/v0.0.1-alpha/notation-v0.0.1-alpha.1-linux-amd64
+  curl -O ~/.docker/cli-plugins/docker-notation https://github.com/notaryproject/notation/releases/download/v0.0.1-alpha/docker-notation-v0.0.1-alpha.1-linux-amd64
+  ```
+
+## Building and Pushing
+- Build and Push the `net-monitor` software
```

Comment:
Same as above, I'd suggest adding a line or two introducing the section before starting the list here.